llama-index-integrations/llms/llama-index-llms-ipex-llm/examples/more_data_type.py

```diff
@@ -33,11 +33,20 @@ def completion_to_prompt(completion):
         choices=["sym_int4", "asym_int4", "sym_int5", "asym_int5", "sym_int8"],
```

Comment:
update the choices to and add GPU related data_types. For a full list of datatypes we can support, refer to https://ipex-llm.readthedocs.io/en/latest/doc/PythonAPI/LLM/transformers.html#automodelforcausallm `load_in_lowbit` param api doc